python/lsst/ts/ess/csc/rpi_data_client.py

```diff
@@ -576,6 +584,40 @@ async def process_hx85ba_telemetry(
         }
         await self.topics.tel_hx85ba.set_write(**telemetry)
 
+    async def process_csat3b_telemetry(
+        self,
+        sensor_name: str,
+        timestamp: float,
+        response_code: int,
+        sensor_data: Sequence[float],
+    ) -> None:
+        """Process the CSAT3B anemometer telemetry and send to EFD.
+
+        Parameters
+        ----------
+        sensor_name : `str`
+            The name of the sensor.
+        timestamp : `float`
+            The timestamp of the data.
+        response_code : `int`
+            The ResponseCode
+        sensor_data : each of type `float`
+            A Sequence of float representing the sensor telemetry data.
+        """
+        device_configuration = self.device_configurations[sensor_name]
+        telemetry = {
+            "sensorName": sensor_name,
+            "timestamp": timestamp,
+            "ux": sensor_data[0],
+            "uy": sensor_data[1],
+            "uz": sensor_data[2],
+            "ts": sensor_data[3],
+            "diagWord": sensor_data[4],
+            "recordCounter": sensor_data[5],
+            "location": device_configuration.location,
+        }
+        await self.topics.tel_airTurbulence.set_write(**telemetry)
```

Comment:
I suggest skipping the dict:
```
await self.topics.tel_airTurbulence.set_write(
    sensorName=sensor_name,
    timestamp=timestamp,
    ux=sensor_data[0],
    uy=sensor_data[1],
    uz=sensor_data[2],
    ts=sensor_data[3],
    diagWord=sensor_data[4],
    recordCounter=sensor_data[5],
    location=device_configuration.location,
)
```
Also, when you do need to construct dicts with keys that are valid identifiers, the dict constuctor can be clearer, e.g.:
```
telemetry = dict(
    sensorName=sensor_name,
    timestamp=timestamp,
    ux=sensor_data[0],
    uy=sensor_data[1],
    uz=sensor_data[2],
    ts=sensor_data[3],
    diagWord=sensor_data[4],
    recordCounter=sensor_data[5],
    location=device_configuration.location,
)
```
as it gets rid of all those double quotes.

Reply:
Great suggestion. I skipped the dicts for all sensors.